### PR TITLE
Quick and Dirty Documentation Improvements

### PR DESCRIPTION
**Pull request by corranwebster**
Turn on Enthought style if available; some clean-up to look better.

Aim is to be included before next release so style can be consistent with other projects.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -77,11 +77,36 @@
...
+                Exception was: {}
+                Enthought Sphinx Theme can be downloaded from
+                https://github.com/enthought/enthought-sphinx-theme"""
+    warnings.warn(RuntimeWarning(msg.format(exc)))
+
+    # Use old defaults if enthought-sphinx-theme not available
+
+    # The name of an image file (within the static path) to place at the top
+    # of the sidebar.
+    html_logo = "e-logo-rev.png"
```
Comment (jwiggins) [inline docs/source/conf.py]: This was `"_static/e-logo-rev.png"` before. Why did it change?

File: docs/source/conf.py [hunk 2]
```diff
@@ -77,11 +77,36 @@
...
+
+    # The name of an image file (within the static path) to use as favicon of
+    # the docs.  This file should be a Windows icon file (.ico) being 16x16
+    # or 32x32 pixels large.
+    html_favicon = "et.ico"
+
+    # The style sheet to use for HTML and HTML Help pages. A file of that name
+    # must exist either in Sphinx' static/ path, or in one of the custom paths
+    # given in html_static_path.
+    html_style = "default.css"
```
Comment (jwiggins) [inline docs/source/conf.py]: This was previously unset

File: docs/source/enable_concepts.rst [hunk 3]
```diff
@@ -90,18 +90,20 @@ Enable :class:`Component` and :class:`Container`.
...
 
 That logic is in :class:`Component`, in the :meth:`\_new_dispatch` method, which
 is called from :meth:`Component.dispatch` (:meth:`\_old_dispatch` will be
 removed in 3.0). If any of these handlers sets event.handled to True, event
 propagation stops. If an event gets as far as the listener tools, then all of
 them get the event.
 
+.. note::
+
   (The notion of an active tool is not used in current code, just older client
```
Comment (jwiggins) [inline docs/source/enable_concepts.rst]: Are the enclosing parentheses necessary now that this is rendered as a proper note?

File: docs/source/conf.py (hunk 1 above)
Comment (corranwebster) [reply docs/source/conf.py]: Because I copy/pasted from a different repo.  I will fix.

File: docs/source/conf.py (hunk 2 above)
Comment (corranwebster) [reply docs/source/conf.py]: No, this is the same as what it was on old line 84 above.

File: docs/source/enable_concepts.rst (hunk 3 above)
Comment (corranwebster) [reply docs/source/enable_concepts.rst]: I was debating that myself.

File: docs/source/conf.py (hunk 2 above)
Comment (jwiggins) [reply docs/source/conf.py]: Ah. Confusing diff. Carry on!

File: docs/source/conf.py (hunk 1 above)
Comment (corranwebster) [reply docs/source/conf.py]: Actually, it looks like the `_static` shouldn't be needed any more - it searches inside the `_static` directory by default.

Comment (mdickinson) [reply docs/source/conf.py]: It doesn't, unfortunately. The path is relative to the directory containing the configuration file, so if `e-logo-rev.png` is in `_static`, then the `_static` prefix is needed.

Doc: http://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_logo

**Pull request merged**